test/keyboards.md

```diff
@@ -24,75 +19,72 @@ showmenu: false
+[keyboards/sil_ipa/](/keyboards/sil_ipa/)
 
-## legacy /keyboards
+[live: keyboards/sil_ipa/](https://keyman.com/keyboards/sil_ipa/)
 
-[_legacy/keyboards](/_legacy/keyboards)
+### keyboards/
 
-[live: _legacy/keyboards](https://keyman.com/_legacy/keyboards)
+[keyboards/](/keyboards/)
 
-[_legacy/keyboards/](/_legacy/keyboards/)
+[live: keyboards/](https://keyman.com/keyboards/)
 
-[live: _legacy/keyboards/](https://keyman.com/_legacy/keyboards/)
+### Old share keyboards/id/share
 
+[keyboards/sil_ipa/share](/keyboards/sil_ipa/share)
 
-### legacy /keyboards/languages/id
+[live: keyboards/sil_ipa/share](https://keyman.com/keyboards/sil_ipa/share)
 
-[_legacy/keyboards/languages/km](/_legacy/keyboards/languages/km)
 
-[live: _legacy/keyboards/languages/km](https://keyman.com/_legacy/keyboards/languages/km)
+### keyboard/{content}
 
-### legacy /keyboards/download
+[keyboard/sil_ipa](/keyboard/sil_ipa)
 
-[_legacy/keyboards/download](/_legacy/keyboards/download?id=sil_ipa&platform=windows&mode=standalone)
+[live: keyboard/sil_ipa](https://keyman.com/keyboard/sil_ipa)
 
-[live: _legacy/keyboards/download](https://keyman.com/_legacy/keyboards/download?id=sil_ipa&platform=windows&mode=standalone)
 
-### legacy /keyboards/legacy
+## Install | Download | Share | bare | .json
 
-[_legacy/keyboards/legacy/id](/_legacy/keyboards/legacy/sil_ipa)
+### keyboards/id.json
 
-[live: _legacy/keyboards/legacy/id](https://keyman.com/_legacy/keyboards/legacy/sil_ipa)
+[keyboards/sil_ipa.json](/keyboards/sil_ipa.json)
 
-### legacy /keyboards/countries/id
+[live: keyboards/sil_ipa.json](https://keyman.com/keyboards/sil_ipa.json)
 
-[_legacy/keyboards/countries/kh](/_legacy/keyboards/countries/kh)
+----
 
-[live: _legacy/keyboards/countries/kh](https://keyman.com/_legacy/keyboards/countries/kh)
+## Search
 
-### legacy /keyboards/id/share
+### keyboards/languages/id
 
-[_legacy/keyboards/sil_ipa/share](/_legacy/keyboards/sil_ipa/share)
+[keyboards/languages/km](/keyboards/languages/km)
 
-[live: _legacy/keyboards/sil_ipa/share](https://keyman.com/_legacy/keyboards/sil_ipa/share)
+[live: keyboards/languages/km](https://keyman.com/keyboards/languages/km)
 
-### legacy /keyboards/id.json
+### keyboards/download.php
 
-[_legacy/keyboards/sil_ipa.json](/_legacy/keyboards/sil_ipa.json)
+[keyboards/download.php?id=sil_ipa](/keyboards/download.php?id=sil_ipa)
 
-[live: _legacy/keyboards/sil_ipa.json](https://keyman.com/_legacy/keyboards/sil_ipa.json)
+[live: keyboards/download.php?id=sil_ipa](https://keyman.com/keyboards/download.php?id=sil_ipa)
 
-### legacy /keyboards/id
+### keyboards/legacy
 
-[_legacy/keyboards/sil_ipa](/_legacy/keyboards/sil_ipa)
+[keyboards/legacy/cameroon querty non-u kbd-fonts](/keyboards/legacy/cameroon%20qwerty%20non-u%20kbd-fonts)
 
-[live: _legacy/keyboards/sil_ipa](https://keyman.com/_legacy/keyboards/sil_ipa)
+[live: keyboards/legacy/cameroon querty non-u kbd-fonts](https://keyman.com/keyboards/legacy/cameroon%20qwerty%20non-u%20kbd-fonts)
```

Comment:
I'm not sure what legacy keyboard ID would exercise these links.

Reply:
legacy could use modern keyboard ids -- it's about old versions of Keyman

Reply:
Hmm - still unclear what should happen. The live site
https://keyman.com/keyboards/legacy/sil_ipa

seems blank too.